test/sanity/validate-modules/main.py

```diff
@@ -1266,8 +1269,8 @@ def _validate_argument_spec(self, docs, spec, kwargs):
                         self.reporter.error(
                             path=self.object_path,
                             code=328,
-                            msg=('Choices value from the documentation (%r) is not compatible '
-                                 'with type %r defined in the argument_spec' % (choice, _type))
+                            msg=("Argument '%s' in documentation defines choices as '%r' "
```

Comment:
Remove the quotes from around `%r` as that may cause a something like `''foo''` to appear in the output.